### DISCUSSION TITLE
How to use the shared .resw files for android and wasm?

### DISCUSSION THREAD

**Discussion by Daantje007** [unoplatform/uno#8387] [Q&A] [answered]
Hi guys, (uno/uwp platform xamarin based vs2022)

How to use the shared .resw files for android and wasm? 
I need a kind of android resource loader or somthing??

This works of course for uwp..
var loader = new Windows.ApplicationModel.Resources.ResourceLoader();
var str = loader.GetString("TitleMainPage");

Thank you,

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. Are you able to provide a repro sample which demonstrates the behavior?

**Reply by Daantje007** [reply]
Hi Jerome Laban,

I have created a simple sample solution, which demonstrates the behavior.
What is the best way to publish for you/all?

Thank you for your time,
D

**Reply by jeromelaban** [reply]
You can attach the sample to this discussion, as a zip file.

**Reply by Daantje007** [reply]
I tried but ist to big i gues..

**Reply by jeromelaban** [reply]
You can delete the bin and obj folders before zipping.

**Reply by Daantje007** [reply]
Jerome Laban,

7z,rar and win is not accepted later i throw a link here afther work... thks

D

**Reply by jeromelaban** [reply]
zip files are accepted.

**Reply by Daantje007** [reply]
[ReproSampleResw.zip](https://github.com/unoplatform/uno/files/8315597/ReproSampleResw.zip)

**Reply by Daantje007** [reply]
its all about this event TxtSorryDontWantToUseUID_Loaded. see the three comments.

**Reply by Daantje007** [reply]
Only UWP works at this point.

**Reply by jeromelaban** [reply]
Thanks for the repro. You're hitting this issue: #5815 and to fix this, you'll need to use:
```csharp
var loader = ResourceLoader.GetForCurrentView();
var str = loader.GetString("Test");
```

**Reply by Daantje007** [reply]
This is great support Jerome respect!  

Where can i donate. i wil donate uno for every question i have haha
And next time ill take a better look at the issues i didnt know about these.

greetings D

**Reply by sasakrsmanovic** [reply]
@Daantje007  - Happy to walk you through our support options if you'd like! Just reach out at sasha@platform.uno.  Pro support is is how we sustain our open source project, a process explained here https://platform.uno/blog/sustaining-the-open-source-uno-platform/
